user-guide/modules/ROOT/pages/discussion-policy.adoc

```diff
@@ -0,0 +1,170 @@
+
+=== Don't Over-quote, Don't Top-Post, and Do Use Inline Replies for Readable Quotations
+
+Prune extraneous quoted text from replies so that only the relevant parts are included. It will save time and make your post more valuable when readers do not have to find out which exact part of a previous message you are responding to.
+
+Don't top-post (where the original message is included verbatim, with the reply above it); _inline_ replies are the appropriate posting style for Boost lists.
+
+The common and very useful inline approach cites the small fractions of the message you are actually responding to and puts your response directly beneath each citation, with a blank line separating them for readability:
+
+Person-you're-replying-to wrote:
+
+```
+> Some part of a paragraph that you wish to reply to goes 
+> here; there may be several lines.
+
+Your response to that part of the message goes here.
+There may, of course, be several lines.
+
+> The second part of the  paragraph that is relevant to your 
+> reply goes here; again there may be several lines.
+
+Your response to the second part of the message goes here.
+```
+
+For more information about effective use of quotation in posts, see https://www.netmeister.org/news/learn2quote.html[How do I quote correctly in Usenet?].
+
+=== Keep the Formatting of Quotations Consistent
+
+Some email and news clients use poor word wrapping algorithms that leave successive lines from the same quotation with differing numbers of leading ">" characters. Microsoft Outlook and Outlook Express, and some web clients, are especially bad about this. If your client offends in this way, please take the effort to clean up the mess it makes in quoted text. Remember, even if you didn't write the original text, it's your posting; whether you get your point across depends on its readability.
+
+The Microsoft clients also create an unusually verbose header at the beginning of the original message text and leave the cursor at the beginning of the message, which encourages users to write their replies before all of the quoted text rather than putting the reply in context. Search online for "Quotefix" and you may find a utility that fixes this issue automatically for your email client.
+
+=== Summarizing and Referring to Earlier Messages
+
+A summary of the foregoing thread is only needed after a long discussion, especially when the topic is drifting or a result has been achieved in a discussion. The mail system will do the tracking that is needed to enable mail readers to display message threads (and every decent mail reader supports that).
+
+If you ever have to refer to single message earlier in a thread or in a different thread then you can use a URL to the message archives. To help to keep those URLs short, you can use https://tinyurl.com/app[tinyurl.com]. Citing the relevant portion of a message you link to is often helpful (if the citation is small).
```

Comment:
I think, we should not recommend using URL shortening services. Those can be a privacy breach, and if they ever stop functioning, the URLs become inaccessible.

Also, it would be helpful to include a link to the page describing mailing lists, which should include links to mailman archives.